ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/proxy/techpreview/deprovision/cucushift-installer-rehearse-gcp-ipi-proxy-techpreview-deprovision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-proxy-techpreview-deprovision
+  steps:
+  - chain: ipi-deprovision
+  - ref: gcp-deprovision-bastionhost
+  - ref: gcp-deprovision-vpc
+  documentation: |-
+    This chain deprovisions all the components created by the
+    cucushift-installer-rehearse-gcp-ipi-proxy-techpreview-deprovision chain.
```

Comment:
Seems here `deprovision` should change to `provision`.

Reply:
done